main.py

```diff
@@ -0,0 +1,67 @@
+
+import sys
+from PyQt5.QtWidgets import QApplication, QDialog
+from PyQt5.QtCore import QTimer
+import gui
+import numpy as np
+
+
+class SilGui(QDialog):
+    def __init__(self, parent=None):
+        super(SilGui, self).__init__(parent)
+        self.ui = gui.Ui_Dialog()
+        self.ui.setupUi(self)
+
+        self.number_of_drones = 5
+
+        self.initialize_plot()
+
+        self.timer = QTimer(self)
+        self.timer.timeout.connect(self.plot)
+
+        self.counter = 5
+        self.timer.start(100)
+
+    def initialize_plot(self):
+        self.data = [self.gen_rand_line(100, 3) for index in range(self.number_of_drones)]
+        self.lines = [self.ui.graphWidget.canvas.ax.plot(dat[0, 0:1], dat[1, 0:1], dat[2, 0:1])[0] for dat in self.data]
+        self.ui.graphWidget.canvas.ax.set_xlim3d([0.0, 10.0])

```

Comment:
all lim3d use the same numbers, this would be a case where I would say that these are magic numbers that should have a variable name associated with it.

Reply:
This file is removed for this PR, because it is not really necessary right now. It will be added later again.